tyssue/core/objects.py

```diff
@@ -142,6 +146,11 @@ def edge_df(self, value):
 
     @property
     def cell_df(self):
+        #- BC -#
+        # Recommends to test if the epithelium has a 'cell' df before returning,
+        # some epitheliums don't require this object (2D)
+        # Should it raise a warning/error if the user asks for
+        # a cell_df from an eptm that doesn't have one ?
```

Comment:
Have you defined an API strategy library-wise for this kind of behavior (user requires something that doesn't exists in the object, let python raise an error or give a library-defined exception) ?